Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/repair_degeneracies.h

```diff
@@ -536,15 +536,94 @@ struct Filter_wrapper_for_cap_needle_removal<TriangleMesh, VPM, Traits, Identity
 
 } // namespace internal
 
-namespace experimental {
-
-// @todo check what to use as priority queue with removable elements, set might not be optimal
+/// \ingroup PMP_repairing_grp
+///
+/// removes almost degenerate faces in a range of faces from a triangulated surface mesh.
+/// Almost degenerated triangle faces are classified as caps or needles: a triangle is said to be a needle<
```

Comment:
Why not calling the function `remove_caps_and_needles()`.

Reply:
Back when the function was created I think we decided to use the other name to be self explanatory. caps/needles is "just" terminology from some papers.

Reply:
Well, that is how you define "almost".